packages/cli/lib/services/dryrun.service.ts

```diff
@@ -166,6 +183,7 @@ class DryRunService {
         const syncRun = new syncRunService({
             integrationService,
             recordsService,
+            dryRunService: new DryRunService(environment, true),
```

Comment:
Sounds weird that a function in `DryRunService` creates a new instance of DryRunService. No?

Reply:
Maybe? Do you see this as blocking? Can refactor a bit if you feel strongly about it.

Reply:
why not passing `this`?

Reply:
because the `returnOutput` instance variable would be different in that case, otherwise we could pass `this`

Reply:
it is confusing to have two level where `.run` is being called. It is not blocking though. up to you